### PR TITLE
Add documentation for Gotify integration

### PR DESCRIPTION
**Pull request by benjmarshall**
## Proposed change

Adding a description for new platform for notify component.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/53050
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2694
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/gotify.markdown [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+1. Setup a Gotify server by following the instructions on their [docs site](https://gotify.net/docs/install). You can run the server using docker or a native Go binary. You must make sure your Gotify server is accessible from the open web for this component to be able to communicate with it.
+2. When you first login to your Gotify server's WebUI setup a new user (unless you want to only use the default admin user), then logout and log back in as your new user.
+3. Add a new application to your Gotify server on the **Apps** tab. Name the new application **Home Assistant**. You can upload the [Home Assistant logo](https://github.com/home-assistant/brands/raw/master/core_integrations/_homeassistant/icon.png) in order to add badges to your notifications.
+4. When your application has been created Gotify will present you with a matching **token**. Copy this now as we will need it in the next step.
+5. Add the Gotify integration to your installation by adding the following to your `configuration.yaml` file:
+
+```yaml
+gotify:
+  token: YOUR_APP_TOKEN
```
Comment (TheZoker) [inline source/_integrations/gotify.markdown]:
```suggestion
  url: YOUR_GOTIFY_URL
  token: YOUR_APP_TOKEN
```

Comment (benjmarshall) [reply source/_integrations/gotify.markdown]: Spot on, i had left out the required URL. Thanks!

Comment (benjmarshall) [reply source/_integrations/gotify.markdown]: Changes included in latest commit as I also needed to add URL item to configuration item table.

File: source/_integrations/gotify.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ha_release: 2021.08
+ha_category:
+  - Notifications
+ha_domain: gotify
+ha_codeowners:
+  - '@benjmarshall'
+ha_iot_class: Cloud Polling
+ha_platforms:
+  - notify
+---
```
Comment (milanmeu) [inline source/_integrations/gotify.markdown:14]:
```suggestion
ha_config_flow: true
---
```

File: source/_integrations/gotify.markdown [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Description
+
+The Gotify integration is a platform for the **notify** component.
+
+This platform allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant on your Android device.
```
Comment (milanmeu) [inline source/_integrations/gotify.markdown]: This also works on other devices.
```suggestion
This integration allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant.
```

File: source/_integrations/gotify.markdown [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ha_platforms:
+  - notify
+ha_config_flow: true
+---
+
+## Description
+
+The Gotify integration is a platform for the **notify** component.
+
+This integration allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant.
```
Comment (frenck) [inline source/_integrations/gotify.markdown]:
```suggestion
This integration allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant.
```

File: source/_integrations/gotify.markdown [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## Setting up
+
+To start getting notifications, you need to follow those simple steps:
```
Comment (frenck) [inline source/_integrations/gotify.markdown]: Simple is subjective and can be condescending for someone that struggles 😉 

```suggestion
To start getting notifications, you need to follow these steps:
```

File: source/_integrations/gotify.markdown [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+ha_config_flow: true
+---
+
+## Description
+
+The Gotify integration is a platform for the **notify** component.
+
+This integration allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant.
+
+## Setting up
```
Comment (frenck) [inline source/_integrations/gotify.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/gotify.markdown [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+4. (Optional) Gotify requires notifications to be assigned to channels called Applications. This integration can configure an Application for you, alternatively you can add a new Application manually to your Gotify server on the **Apps** tab. You can upload the [Home Assistant logo](https://github.com/home-assistant/brands/raw/master/core_integrations/_homeassistant/icon.png) in order to add badges to your notifications. The new Application will an **app token** associated with it which you can view by clicking the eye symbol. If you want to use a manually created Application you will need need this **app token** in the next step. If you don't want to manually create an Application this integration will create an Application called Home Assistant for you in the next step.
+5. This integration uses a Config Flow. Visit the Integrations page of your Home Assistant and select **Add Integration**. Enter the host URL of your Gotify server, the **client token** generated in step 3, and if desired the **app token** generated in step 4.
```
Comment (frenck) [inline source/_integrations/gotify.markdown]: This step needs to be done using our standard include:

```
{% include integration/config_flow.md %}
```

File: source/_integrations/gotify.markdown [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+6. When the Config Flow wizard is complete you should see a new entry for the Gotify integration. By default this will be called gotify_home_assistant, but this may be different if an Application was created manually by following step 4.
```
Comment (frenck) [inline source/_integrations/gotify.markdown]: Step 6 doesn't really add something. Additionally, names are not guaranteed.

File: source/_integrations/gotify.markdown [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+7. Install the [Gotify Android App](https://github.com/gotify/android) and configure it to connect to your Gotify server using your user's details. It's also a good idea to [follow the instructions](https://github.com/gotify/android#disable-battery-optimization) to disable battery optimisations.
```
Comment (frenck) [inline source/_integrations/gotify.markdown]: I think this is more a thing for the Gotify documentation.

File: source/_integrations/gotify.markdown [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+8. Now you can use your new Gotify integration entry as a platform for your **notify** service. A new service will be available as notify.***name***, e.g. notify.gotify_home_assistant. The new service can be tested using the Developer Tools, or integrated straight into an automation/script.
```
Comment (frenck) [inline source/_integrations/gotify.markdown]: This is not really part of the setup, if you want to provide examples, please do so below, but not here.

File: source/_integrations/gotify.markdown [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+### That's it!
+
+Now you can use your new Gotify service inside your Home Assistant to send any notifications or alerts.
+
```
Comment (frenck) [inline source/_integrations/gotify.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by benjmarshall** [comment]
@frenck Thanks for the review. I've updated based on your suggestions and have included the standard config flow section. Hopefully this meets the standards now. Anything else, please add further comments.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**